Brain/brain_utils.h

```diff
@@ -132,7 +132,7 @@ int decode_received_packet(String packet, valveInfo *valve) {
   }
   int action = packet.substring(data_start_index + 1,data_end_index).toInt();
   
-  String checksumstr = packet.substring(data_end_index + 1, packet.length()-2);
+  String checksumstr = packet.substring(data_end_index + 1, packet.length()-1);
```

Comment:
The reason it was -2 previously was because the packet was something like {1,2,3|5} so we didn't want the last '}' in the checksum. I believe this should stay -2 instead of -1 unless there was something changed about the packet format.

Reply:
That's what I thought too, but we were getting some errors this last weekend where the last value of the checksum was being chopped off. Will take note of this and watch out for it this saturday

Reply:
Worked perfectly with -1. 
makes sense since substring is exclusive of the end index
![image](https://user-images.githubusercontent.com/10129077/99325916-1c18ee00-282c-11eb-927c-89cc566e4997.png)

Reply:
Worked fine this weekend; this makes sense too since substring is exclusive of end index
![image](https://user-images.githubusercontent.com/10129077/99326312-f213fb80-282c-11eb-88ed-6d98716c537a.png)